src/common.ts

```diff
@@ -322,6 +320,10 @@ function getInitializationOptions(serverMode: ServerMode, context: ExtensionCont
         .get<Record<string, string>>('vueserver.json.customBlockSchemaUrls'),
     },
     additionalExtensions: additionalExtensions(),
+    semanticTokensLegend: {
+      tokenTypes: ['component'],
```

Comment:
Hi @yaegassy 👋, this config may not have an effect for coc-volar because it depends on client semanticTokenScopes, in VSCode it in https://github.com/johnsoncodehk/volar/blob/b66b64f579b5cafee9b8d70aa9f9303f39b6df49/extensions/vscode-vue-language-features/package.json#L121-L146.

Reply:
Hi! Thanks for explaining. 🙇 I'll remove the semanticTokensLegend in the initializationOptions in the next coc-volar.